conf/ocsci/odf-live-must-gather.yaml

```diff
@@ -0,0 +1,5 @@
+---
+# This config file can be used to enforce live must gather image for ODF
+REPORTING:
+  ocs_must_gather_image: "registry.redhat.io/odf4/ocs-must-gather-rhel8"
+  ocs_must_gather_latest_tag: 'latest'
```

Comment:
Should we also change the name of the keys from ocs to odf?

Reply:
nope, this is for usecase if you just want to enforce the live image for must gather, we are still taking it from this key in the code.

The logic for --live-deploy parameter is taken from {odf_ocs}_live_must_gather_image which is in default config.

We do not rename usually values from OCS to ODF when code already using this naming convention.

Reply:
Ack